src/pd/client.rs

```diff
@@ -480,4 +480,31 @@ impl PdClient for RpcClient {
     fn handle_reconnect<F: Fn() + Sync + Send + 'static>(&self, f: F) {
         self.leader_client.on_reconnect(Box::new(f))
     }
+
+    fn get_gc_safe_point(&self) -> PdFuture<u64> {
+        let timer = Instant::now();
+
+        let mut req = pdpb::GetGCSafePointRequest::new();
+        req.set_header(self.header());
+
+        let executor = move |client: &RwLock<Inner>, req: pdpb::GetGCSafePointRequest| {
+            let option = CallOption::default().timeout(Duration::from_secs(REQUEST_TIMEOUT));
+            let handler = client
+                .rl()
+                .client
+                .get_gc_safe_point_async_opt(&req, option)
+                .unwrap();
```

Comment:
`unwrap` will cause TiKV panic when pd leader is killed and new leader hasn't elected. I think we should tolerate this case.

Reply:
I was just following how other functions here do. If there's problem using `unwrap` here, other functions here must have this problem too.